docs/en/03-benchmark/benchmark.md

```diff
@@ -1641,6 +1641,18 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">-</td>
     <td align="center">-</td>
   </tr>
+  <tr>
```

Comment:
could add to regression test config for easy test: https://github.com/open-mmlab/mmdeploy/blob/master/tests/regression/mmrotate.yml